tools/Ecoregionalization_workflow/BRT_model.xml

```diff
@@ -0,0 +1,110 @@
+<tool id="brt_analysis" name="BRT tool prediction" version="0.1.0+galaxy0" >
+    <description>for species distribution modelling</description>
+    <requirements>
+       <requirement type="package" version="4.3.0">r-base</requirement>
+       <requirement type="package" version="1.3_14">r-dismo</requirement>
+       <requirement type="package" version="2.1.8">r-gbm</requirement>
+       <requirement type="package" version="3.4.2">r-ggplot2</requirement>
+    </requirements>
+    <command detect_errors="exit_code"><![CDATA[
+        Rscript
+         '$__tool_directory__/brt.R'
+         '$enviro'
+         '$species_files'
+         '$input3'
+         '$output1'
+         '$output2'
+         '$output3'
+         '$output4'
+    ]]></command>
+    <inputs>
+      <param name="enviro" type="data" format="txt,csv,tsv" label="Environment Data"/>
+      <param type="data" name="species_files" label="Occurrences Data File" format="tabular,txt,csv" multiple="True" />
+      <param name="input3" type="text" label="write the name of your abiotic parameters (comma separated)"/>
```

Comment:
How is a user supposed to know the options? Are they free text? Or can we offer a select list?

Reply:
Are those parameters from the header of tabular files? I wonder if `data_column` can be used here https://docs.galaxyproject.org/en/latest/dev/schema.html#data-column

Reply:
THANK YOU Björn! I think we can use this parameter type as coming from the header of the tabular file! Can you inform me this will give exactly the same kind of string? For the related test tag, I have to list the column numbers as a list like "c3,c4,c5...." ?

Reply:
I think I am done, "jsut" this point who is important to look at. Can you inform me if `data_column` is writing a character string exactely as we are making actually ? Like `Carbo,Grav,Maxbearing` ? THANK you so much for your help !